### PR TITLE
Add Node/NPM mini-lesson and Open-Zeppelin fundamentals

### PR DESCRIPTION
**Pull request by losMacos**
- Mini-lesson on installation of Node.js/NPM via NVM for Windows-WSL2, MacOS and Ubuntu Linux
- Overview of the utility and prominence of Open Zeppelin audited contracts

### CODE REVIEW

**Comment by with-heart** [comment]
When writing Markdown, paragraphs are handled automatically by line breaks and the lines will wrap based on the width of the containing element. You'll rarely ever need to use the `br` tag!

**Comment by georgemac510** [comment]
Thanks @with-heart We've cleaned up all of the `<br>` tags and it deploys successfully after that fix.

File: lessons/1.mdx [hunk 1]
```diff
@@ -74,7 +74,7 @@ There are a lot more features in Remix, but those are the ones we’ll be workin
 
 To get started we will use the Remix IDE to write, compile, test and debug code quickly and easily. Head over to [https://remix.ethereum.org/](https://remix.ethereum.org/) to see what it’s all about and get started. 
 
-![Remix IDE Toolbar](/assets/lessons/1/img_1.png)
+![Remix IDE Toolbar](/public/assets/lessons/1/img_1.png)
```
Comment (PBillingsby) [inline lessons/1.mdx:77]: These additions of `public/` seem to have broken the rendering of those images.

Comment (losMacos) [reply lessons/1.mdx:77]: I think it actually fixed them if you check the `fix` at  https://github.com/losMacos/school-of-code/blob/fundamentals/lessons/1.mdx compared to the original at https://github.com/Developer-DAO/school-of-code/blob/main/lessons/1.mdx

**Comment by georgemac510** [comment]
Broken into several smaller PRs that have been merged.

**Pull request closed without merging**